### PR TITLE
fix(NcAppNavigationItem): align utils with actions and other components

### PR DESCRIPTION
**Pull request by GVodyanov**
### ☑️ Resolves

- Calendar sidebar alignment, no issue

### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
Screenshot from 2024-09-05 13-58-56 | Screenshot from 2024-09-05 13-58-07

### CODE REVIEW

File: src/assets/NcAppNavigationItem.scss [hunk 1]
```diff
@@ -170,7 +170,7 @@
 	/* counter */
 	.app-navigation-entry__counter-wrapper {
 		// Add slightly more space to the right of the counter
```
Comment (GretaD) [inline src/assets/NcAppNavigationItem.scss:172]: this will change the counter space, did you test it with the counter? if it looks off?

Comment (susnux) [reply src/assets/NcAppNavigationItem.scss:172]: Bildschirmfoto_20240905_160729

2 and 4 have a counter, 3 has actions

**Comment by ShGKme** [comment]
Don't forget to add `bug`/`enhancement` tags and follow conventional commit style. It is used in the changelog generation

File: src/assets/NcAppNavigationItem.scss (hunk 1 above)
Comment (GVodyanov) [reply src/assets/NcAppNavigationItem.scss:172]: Screenshot from 2024-09-05 16-39-13

I would argue it looks better now though

Comment (GVodyanov) [reply src/assets/NcAppNavigationItem.scss:172]: image

If they are moved more to the right, the actions become misaligned with `NcAppNavigationCaption`, so unless you have any better ideas @susnux I would keep it the way it is currently

Comment (susnux) [reply src/assets/NcAppNavigationItem.scss:172]: That was no critic, just wanted to add screenshots for Gretas comment, I am fine with any of that :)

**Review by susnux — APPROVED** [review]

**Review by GretaD — APPROVED** [review]

**Pull request merged**